api/v1alpha1/releaselink_webhook.go

```diff
@@ -51,12 +55,20 @@ var _ webhook.Validator = &ReleaseLink{}
 
 // ValidateCreate implements webhook.Validator so a webhook will be registered for the type.
 func (r *ReleaseLink) ValidateCreate() error {
-	return r.isTargetValid()
+	err := r.isTargetValid()
+	if err != nil {
+		return err
+	}
```

Comment:
As seen elsewhere in the code, Go allows for a [short statements to be placed ahead of the conditional expression](https://go.dev/ref/spec#If_statements).
Something like:
```
if err := r.isTargetValid() ; err != nil {
    return err
}
```
It may seem trivial, but it shaves off one line, here, another there, and it adds-up.
The only potential gotcha is `err` becomes limited to the scope of `if`'s curly brace, but I dont' see that being a problem here.

Note: this is not asking for code changes, just something I do... feel free to ignore.

Reply:
Although I agree for this particular case, I actually have more concerns about the auxiliary functions themselves. I missed this when `isTargetValid` was added but the name itself needs an improvement. That name suggests that the function returns a boolean, which is not the case.

Unless there's a better alternative, let's rename the functions to `validateTarget` and `validateAutoReleaseLabel`.

Reply:
I changed it to be the shorter syntax and renamed the functions